docs/config.rst

```diff
@@ -423,6 +423,8 @@ storageSubPath      The path in the persistent volume to be mounted (default: ro
 computeResourceType Define whether use Kubernetes ``Pod`` or ``Job`` resource type to carry out Nextflow tasks (default: ``Pod``).
 fetchNodeName       If you trace the hostname, activate this option (default: ``false``, requires version ``22.05.0-edge`` or later).
 volumeClaims        (deprecated)
+requestHttpReadTimeoutInMilliSeconds    Defines the Kubernetes client request http connection read timeout (default 60000ms)
+requestHttpConnectionTimeoutInMilliSeconds  Defines the Kubernetes client request http connection timeout (default 60000ms)
```

Comment:
```suggestion
httpConnectionTimeout  Defines the Kubernetes client request HTTP connection timeout e.g. ``'60s'``'
```

Nextflow usually uses [Duration](https://github.com/nextflow-io/nextflow/blob/84d5a747d1ac4baa425313f8e16e0c3efa6a642b/modules/nf-commons/src/main/nextflow/util/Duration.groovy#L35-L35) class for those setting